### PR TITLE
L1 volume loss (replace MSE for volume nodes)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The surface curriculum transitions from MSE→L1 and has been our biggest win. The volume loss is still pure MSE. In prior rounds, L1 volume loss was tested but on a different codebase. With our current optimized setup (lr=2e-3, T_max=10, dual heads, surface curriculum), L1 for volume may now compound well — it prevents the backbone from over-focusing on volume outliers, freeing capacity for surface features.

**Key difference from PR #282 (vol curriculum):** That PR used the MSE→L1 curriculum for volume. This uses flat L1 from the start — the backbone has always benefited from L1 in the surface loss, so giving it L1 on volume too should be consistent.

## Instructions

In `train.py`, change the volume loss (line 146):

```python
# OLD:
        vol_loss = (sq_err * vol_mask.unsqueeze(-1)).sum() / vol_mask.sum().clamp(min=1)

# NEW:
        vol_err = (pred - y_norm).abs()
        vol_loss = (vol_err * vol_mask.unsqueeze(-1)).sum() / vol_mask.sum().clamp(min=1)
```

**Keep surface loss curriculum unchanged.** Run:
```bash
python train.py --agent fern --wandb_group mar14b-l1-vol --wandb_name "fern/l1-vol"
```

## Baseline
| Metric | Value |
|--------|-------|
| **surf_p** | **107.35** |
| surf_Ux | 1.23 |
| surf_Uy | 0.84 |

---

## Results

**W&B run**: `qqpotq93`
**Epochs completed**: 10 (wall-clock ~5.4 min)
**Best epoch**: 10
**Peak VRAM**: ~15.4 GB

| Metric | This run | Baseline | Delta |
|---|---|---|---|
| val/loss | 2.9116 | 2.45 | +0.46 |
| Surface MAE Ux | 1.19 | 1.23 | -0.04 |
| Surface MAE Uy | 0.85 | 0.84 | +0.01 |
| Surface MAE p | 117.44 | **107.35** | **+10.09 (+9.4%)** |
| Volume MAE Ux | **5.16** | 5.44 | -0.28 |
| Volume MAE Uy | 2.56 | 2.57 | -0.01 |
| Volume MAE p | 153.0 | 152.5 | +0.5 |

**What happened**: L1 volume loss hurt surf_p significantly (107.35 → 117.44, +9.4%). While vol_Ux improved somewhat, the primary metric regressed. The MSE volume loss appears to be important for pressure accuracy — switching to L1 for volume reduces the gradient signal for large pressure errors in the volume, which may degrade the shared backbone representations that the surface decoder relies on. The surface and volume losses are not independent; the volume MSE helps establish strong pressure field representations that benefit surface accuracy.

**Suggested follow-ups**:
- MSE volume loss is confirmed better for surf_p. Do not change it.
- Try a volume curriculum (MSE→L1 for volume with alpha=4) to get some benefit from L1 without sacrificing early MSE pressure learning.

### CODE REVIEW

**Comment by tcapelle** [comment]
Closing — L1 volume loss gives surf_p=117.4, worse than baseline 107.35. MSE for volume remains better.
